src/views/scheduleMessage.js

```diff
@@ -1,13 +1,14 @@
 import './scheduleMessage.css';
 
-import MessageInput from '../components/messageInput/messageInput'
+import MessageInput from '../components/messageInput/messageInput';
+import TimePicker from '../components/timepicker/timePicker'
 import { useContext, useEffect, useRef, useState } from 'react';
 import AppContext from '../AppContext';
 
 function ScheduleMessage() {
     const {slackClient, workspace, isLoading} = useContext(AppContext);
     const [scheduledMessages, setScheduledMessages] = useState([]);
-    const timeInput = useRef(null)
+    const [timeInput, setTimeInput] = useState(null)
```

Comment:
Nit: rename to timestamp
```suggestion
    const [setTimestamp, setTimestamp] = useState(null)
```

Reply:
Other places would need to be updated as well